src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -128,6 +128,7 @@ public sealed partial class Server : IDisposable {
                                             @params.initializationOptions.includeFiles,
                                             @params.initializationOptions.excludeFiles,
                                             _services.GetService<IIdleTimeService>());
+            _indexManager.IndexWorkspace(cancellationToken).DoNotWait();
```

Comment:
I'm not sure that this `cancellationToken` is the right one to use, unfortunately. I believe this would be cancelled once the initialization occurs.